### PR TITLE
OCP4: Add rules for SRG-APP-000456-CTR-001125

### PR DESCRIPTION
**Pull request by jhrozek**
#### Description:

The rules make sure that there are allowed registries set for both pulls
and imports and that all imagestreams check for updates.

#### Rationale:

OCP4 STIG

#### Review Hints:

- I will post a link to the SRG export later

### CODE REVIEW

File: controls/srg_ctr/SRG-APP-000456-CTR-001125.yml [hunk 1]
```diff
@@ -5,4 +5,8 @@ controls:
   title: '{{{ full_name }}} registry must contain the latest images with most
     recent updates and execute within the container platform runtime as authorized
     by IAVM, CTOs, DTMs, and STIGs. '
-  status: pending
+  status: automated
+  rules:
+    - imagestream_sets_schedule
+    - ocp_allowed_registries
+    - ocp_allowed_registries_for_import
```
Comment (Mab879) [inline controls/srg_ctr/SRG-APP-000456-CTR-001125.yml:12]: Pleasure ensure end this file with a newline. The check eof script will be updated to catch this in the future.

Comment (jhrozek) [reply controls/srg_ctr/SRG-APP-000456-CTR-001125.yml]: thanks for catching this and even more for working on automation

**Comment by jhrozek** [comment]
The STIG tests passed - @rhmdnd, @Vincent056 PTAL

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Comment by rhmdnd** [comment]
I imagine the YAML lint failures are a false positive because we're not using ansible here?

**Comment by jhrozek** [comment]
> I imagine the YAML lint failures are a false positive because we're not using ansible here?

Looks like this is actually a bug, I created https://github.com/ComplianceAsCode/content/issues/10483 after talking to @ggbecker

**Comment by rhmdnd** [comment]
> > I imagine the YAML lint failures are a false positive because we're not using ansible here?
> 
> Looks like this is actually a bug, I created #10483 after talking to @ggbecker

Thanks! I'll rebase and pick up the fix in https://github.com/ComplianceAsCode/content/pull/10484

**Comment by jhrozek** [comment]
> > > I imagine the YAML lint failures are a false positive because we're not using ansible here?
> > 
> > 
> > Looks like this is actually a bug, I created #10483 after talking to @ggbecker
> 
> Thanks! Looks like we can rebase to pick up the fix in #10484 ?

Sure done. No other changes. I'll also kick the STIG e2e tests just to be sure.

**Comment by jhrozek** [comment]
Since this was just a rebase and the patch was lgtmed earlier, I'm going to merge.

**Pull request merged**